modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -343,30 +373,50 @@ function AddFDSFilterModalContent({
 			</ClayModal.Header>
 
 			<ClayModal.Body>
-				<ClayForm.Group>
-					<label htmlFor={nameFormElementId}>
-						{Liferay.Language.get('name')}
-
-						<span
-							className="label-icon lfr-portal-tooltip ml-2"
-							title={Liferay.Language.get(
-								'if-this-value-is-not-provided,-the-name-will-default-to-the-field-name'
-							)}
-						>
-							<ClayIcon symbol="question-circle-full" />
-						</span>
-					</label>
-
-					<ClayInput
-						aria-label={Liferay.Language.get('name')}
-						name={nameFormElementId}
-						onChange={(event) => setName(event.target.value)}
-						placeholder={
-							selectedField?.label || Liferay.Language.get('name')
-						}
-						value={name}
-					/>
-				</ClayForm.Group>
+				{Liferay.FeatureFlags['LPS-172017'] ? (
+					<ClayForm.Group>
+						<InputLocalized
+							error={errorMessage}
+							id={nameFormElementId}
+							label={Liferay.Language.get('name')}
+							name="name"
+							onChange={(newFieldLabel) => {
+								setI18nFilterNames({
+									...i18nFilterNames,
+									...newFieldLabel,
+								});
+							}}
+							required
```

Comment:
See how I implemented frontend validation in https://github.com/liferay-frontend/liferay-portal/pull/3659. It's based on `onBlur` events, enabling save button when entire form passes validation. If it has flaws, we can tweak it. We can even discuss a different behavior, that might have better UX. But in any case, we should implement the same solution in both places.